components/lsp-form/useTokensCreatedEvents.ts

```diff
@@ -0,0 +1,69 @@
+import { useQuery } from "react-query";
+import { ethers } from "ethers";
+import assert from "assert";
+
+export default function useTokensCreatedEvents(
+  contract: ethers.Contract | null,
+  address: string | null
+) {
+  const { data, error, refetch } = useQuery<TokensCreated[] | undefined | void>(
+    "rewardsRetrievedEvents",
+    () => {
+      return queryTokensCreatedEvents(contract, address)
+        .then((res) => res)
+        .catch((err) => console.log(err));
+    },
+    { enabled: contract !== null && address !== null }
+  );
+
+  return { data, error, refetch };
+}
+
+/*
+  event TokensCreated (
+    address indexed sponsor,
+    uint256 indexed collateralUsed,
+    uint256 indexed tokensMinted,
+  );
+*/
```

Comment:
Comment?

Reply:
I am leaving this as a comment as to the shape of the events from the contract -- documentation. I can delete if you'd like though.

Reply:
Gotcha -- makes sense!